src/browser/api_protocol/api_handlers/webcontents.ts

```diff
@@ -49,6 +51,28 @@ function executeJavascript(identity: Identity, message: APIMessage, ack: Acker,
 
     return nack(new Error('Rejected, target window is not owned by requesting identity'));
 }
+
+function findInPage(identity: Identity, message: APIMessage, ack: Acker): void {
+    const { payload } = message;
+    const { searchTerm, options } = payload;
+    const dataAck = Object.assign({}, successAck);
+    const windowIdentity = getTargetWindowIdentity(payload);
```

Comment:
Question: Is there a reason we wouldn't want to use the `identity` here instead of the `payload` identity? Probably doesn't make a functional difference tho

Reply:
All the other functions are using the payload one, didn't see a need to change that.